roles/gerrit/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+---
+- name: Install Java and python-bugzilla
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - java-1.8.0-openjdk
+  - python-bugzilla
+  - postgresql
+  - git
+
+- name: Create a user for review
+  user:
+    name: "{{gerrit_user}}"
+    home: "{{gerrit_user_home}}"
+
+- name: Setup the selinux boolean for proxying
+  seboolean:
+    name: httpd_can_network_connect
+    state: yes
+    persistent: yes
+
+- name: Check if Gerrit is installed
+  stat:
+    path: "{{gerrit_user_home}}/{{gerrit_home}}"
+  register: gerrit_status
+
+- name: "Fail when Gerrit is not setup"
+  fail:
+    msg: "Please setup Gerrit at {{gerrit_user_home}}/{{gerrit_home}}"
+  when: gerrit_status.stat.exists == False
+
+- name: Setup the gerrit.config file
+  template:
+    src: gerrit.config.j2
+    dest: "{{gerrit_user_home}}/{{gerrit_home}}/etc/gerrit.config"
+    owner: "{{gerrit_user}}"
+    mode: 0755
+  notify: restart gerrit
+
+- name: Setup the apache config file
+  template:
+    src: reverse_proxy.conf.j2
+    dest: "/etc/httpd/conf.d/{{website_domain}}.conf.d/reverse_proxy.conf"
+    mode: 0744
+  notify: verify config and restart httpd
+
+- name: Setup the replication.config file
+  template:
+    src: replication.config.j2
+    dest: "{{gerrit_user_home}}/{{gerrit_home}}/etc/replication.config"
+    owner: "{{gerrit_user}}"
+    mode: 0755
+  when: gerrit_replication is defined
```

Comment:
Might be worthwhile refactor in a separate file and then use include + when on it.